src/modules/missionHelper/i18n/nb_NO.json

```diff
@@ -65,6 +65,7 @@
 			"swat_suv": "Panstret kjøretøy | Pansdrede kjøretøy",
 			"traffic_car": "Utrykningspolitibil | Utrykningspolitibiler",
 			"water_tankers": "Tankbil | Tankbiler"
+			"diver_car": "Dykkerbil | Dykkerbiler"
```

Comment:
If you want to add this, you could change the already existing "diver_car" to "diver_units". And please create the PR to Dev next time.